bedrock/settings/__init__.py

```diff
@@ -246,8 +246,10 @@
 csp_extra_frame_src = config("CSP_EXTRA_FRAME_SRC", default="", parser=ListOf(str))
 if csp_extra_frame_src:
     _csp_child_src = list(set(_csp_child_src + csp_extra_frame_src))
+csp_report_uri = config("CSP_REPORT_URI", default="") or None
 
 CONTENT_SECURITY_POLICY = {
+    "REPORT_PERCENTAGE": config("CSP_REPORT_PERCENTAGE", default="100", parser=int),
```

Comment:
Looks like Dev is set to report 5% and stage 2%, but prod has no CSP_REPORT_PERCENTAGE set, which would default to 100% on rollout and could spam Sentry quite hard, maybe?

Might be worth making `default="1"` here, as well as setting a real value for prod in `webservices-infra`